src/apps/main/watchface_digital.cpp

```diff
@@ -35,14 +35,23 @@ void drawDate(OswHal* hal) {
 
     hal->gfx()->print(date_Array);
     hal->gfx()->print(" ");
-    hal->gfx()->printDecimal(monthInt, 2);
-    hal->gfx()->print("/");
-    hal->gfx()->printDecimal(dayInt, 2);
+
+    //i really would want the date to be dynamic based on what's in the config, but the most efficient thing to do right now is simply two if statements covering the 2 common conditions.
+    if (DATE_FORMAT == "mm/dd/yyyy") {
```

Comment:
can we change this that it still works when `DATE_FORMAT` is not defined?

I've seen a couple of users stumbling across not updating their `config.h` respectively.

If we have a default case for `TIME_FORMAT` and `DATE_FORMAT` not being defined in addition would be optimal.

Thanks you :)

Reply:
Alright, both TIME_FORMAT and DATE_FORMAT should be protected in current uses against being undefined